File: Detectando_oleo_no_oceano.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main(){
    vector<int> Areas; 
    int n;
    cin >> n;
    for(int i=0;i<n;i++){
        int valor;
        cin >> valor;
        Areas.push_back(valor);
    }
    int contagem = count(Areas.begin(), Areas.end(), 1);
    int res = (contagem*100)/n;
    if(res<30){
        cout << "Regiao segura";
    }
    else if (res>=30 && res<=50)
    {
        cout << "Regiao em estado de alerta";
    }
    else
    {
        cout << "Regiao com alto indice de perda de biodiversidade";
    }
    
}

File: detectando_oleo_no_oceano/Detectando_oleo_no_oceano.py
n = int(input())
lista = []
i=0
while i < n:
    i+=1
    entrada = int(input())
    lista.append(entrada)
contagem = lista.count(1)
res = (contagem*100)/n
if res<30:
    print("Regiao segura")
    
elif res>=30 and res<=50:
    print("Regiao em estado de alerta")
    
else:
    print("Regiao com alto indice de perda de biodiversidade")
    